polylogue/cli/index_cli.py

```
@@ -0,0 +1,54 @@
+
+def run_index_check(args: argparse.Namespace, env) -> None:
+    report: Dict[str, Any] = {
+        "sqlite": {"status": "ok", "messages": []},
+        "qdrant": {"status": "skipped", "messages": []},
+    }
+    try:
+        notes = verify_sqlite_indexes(attempt_rebuild=bool(getattr(args, "repair", False)))
+        report["sqlite"]["messages"].extend(notes)
+        if notes:
+            report["sqlite"]["status"] = "updated"
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)